src/registrar/tests/test_views.py

```diff
@@ -1010,20 +998,6 @@ def test_domain_detail_profile_feature_on(self):
         self.assertContains(response, "Your profile")
         self.assertNotContains(response, "Your contact information")
 
-    @less_console_noise_decorator
-    def test_domain_your_contact_information_when_profile_feature_off(self):
```

Comment:
Removed test_domain_your_contact_information_when_profile_feature_off and test_domain_your_contact_information_when_profile_feature_on since we are removing Your contact information page which relies on submitter data